server/start.go

```diff
@@ -476,6 +391,94 @@ func genPvFileOnlyWhenKmsAddressEmpty(cfg *config.Config) *pvm.FilePV {
 	return nil
 }
 
+func startGrpcServer(
```

Comment:
cosmos/cosmos-sdk#16152

Reply:
I tried to minimize changes. But it's probably better to apply full refactoring this time. I will apply this PR as well.

Reply:
You may state your intention, and create an issue for the later refactoring. However, no one will stop you if you want to apply it right now.